articles/active-directory-b2c/date-transformations.md

```diff
@@ -206,3 +206,40 @@ To run the claim transformation, you first need to get the current dateTime and
     - **timeSpanInSeconds**: 7776000 (90 days)
 - Output claims:
     - **result**: true
+
+## IsTermsOfUseConsentRequired
+
+Determine whether the terms of use updated date is newer than the terms of use consented date. The result is a new boolean ClaimType with a value of `true` or `false`.
+
+| Item | TransformationClaimType | Data Type | Notes |
+| ---- | ----------------------- | --------- | ----- |
+| InputClaim | termsOfUseConsentDateTime | dateTime | The dateTime of when the user has previously consented to terms of use. Undefined value returns `True` result. |
```

Comment:
Please change to: The dateTime to check. Undefined value returns `True` result.

Reply:
updated 👍 Thanks